src/tap/include/tap/tapcell.h

```diff
@@ -97,85 +115,108 @@ class Tapcell
   void reset();
   int removeCells(const std::string& prefix);
 
+  void insertBoundaryCells(const BoundaryCellOptions& options);
+  void insertTapcells(const Options& options);
+
  private:
-  struct FilledSites
+  enum class EdgeType
   {
-    int yMin = 0;
-    int xMin = 0;
-    int xMax = 0;
+    Left,
+    Top,
+    Right,
+    Bottom,
+    Unknown
```

Comment:
How is this used?

Reply:
This is used to annotate the edges when I'm finding the boundaries of the rows areas.

Reply:
I was asking about the Unknown value

Reply:
same as the corrners answer.